lib/bindings/MQTTBinding.js

```diff
@@ -123,21 +123,23 @@ function unsubscribeAll(callback) {
  * Start the binding.
```

Comment:
An entry at CHANGES_NEXT_RELEASE describing the change introduced by this PR should be added.